src/amltk/__init__.py

```diff
@@ -77,4 +82,7 @@
     "Trial",
     "TxtLoader",
     "YAMLLoader",
+    "MatchDimensions",
+    "MatchChosenDimensions",
+    "build_model_from_pipeline",
```

Comment:
So `pytorch` is an optional install, so I'm not sure you can include `build_model_from_pipeline` here. A good way to test that all pytorch code is hidden behind an explicit import is to uninstall pytorch and run one of the other examples.

Reply:
Should I remove it from the global namespace?

Reply:
Aye, I've done the same with scikit-learn as well, such that you shouldn't need scikit-learn installed to use `pytorch` things